blogposts/2021/better-onboarding-advice-from-3-engineering-leaders.md

```diff
@@ -0,0 +1,127 @@
+> Some people have opted for a lot of sessions, and come with a list of questions, and that's how they prefer to learn. 
+> Others prefer to find information asynchronously through Slack or Confluence or whatnot."
+
+Experience level also affects how new hires onboard. Junior engineers likely need more guidance and also reassurance about what is a "normal pace" of onboarding. 
+That's not to discount the benefit of reassurance for senior engineers, as well, but it's customary to expect a senior engineer starts directing their own onboarding after getting through the standard stuff.
+
+## Offer emotional support
+
+Even senior engineers can suffer from impostor syndrome. Literally everyone feels intimidated when diving into a new large codebase. 
+Finding the right balance between past experience and new territory is crucial, and organizations that value this should go out of their way to create a welcoming culture and environment.
+
+For new hires, having a marker for "am I living up to expectations" in the first couple of weeks can ease their worries. 
+It's often better to err on the side of having an easier onramp, due to all the pitfalls of onboarding. 
+This also allows an EM to establish a baseline for how a new hire works and what they're capable of. 
+If you overshoot, you'll be unsure if it's a performance issue or if the person just happened to fail due to tough circumstances.
+
+Remember, you hired someone because you thought they were excellent. So if they are struggling, you should figure out what might be throwing them off, or revisit your hiring processes if this is happening often. 
+It's common for fast-growing startups to experience some churn and attrition, but if this is happening a lot, it's important for someone in the org to be asking these questions and sharing as much learning as they can while still respecting people's individual privacy about employment decisions.
+
+## Embrace continuous onboarding
+
+> "I think most people look at onboarding in the same way we look at education: you go to school, you graduate, and you're done with learning. 
+> But the reality is you should never be done with learning. And in the same way, when it comes to onboarding onto codebases and infrastructure and that kind of thing, you should never be done with learning there because those systems aren't in a consistent state; they're evolving over time as the business evolves." – Ryan
+
+This is especially true when a developer switches teams or starts a new project that involves a previously unfamiliar part of the codebase. 
+There are some things they already know and don't need to repeat—for example, they likely already have the dev environment set up and have the big picture of the overall company—but there is still a lot of context to acquire. 
```

Comment:
@rebeccadee WDYT about switching the POV to first-person plural? When talking about developers, use "we" instead of "they," because I, the author, identify as a developer. When referring to EMs, continue to use "they".

Reply:
@beyang I went back and forth on this a bit. I understand it feels awkward for you to refer to devs as "they" if you identify as one, but we are addressing EMs more than we are individual devs in this post, so I think the way it's written will feel more natural to most people reading it.